SupportClient/SupportClient/Controllers/ViewController.swift

```diff
@@ -0,0 +1,198 @@
+        
+    var feedbackData: [Feedback]!
+    var filteredFeedbackData = [Feedback]() // TODO: For later
+    
+    var countEditTaps: Int = 0
+    var isFeedback: Bool!
+    let cellHeight: CGFloat = 75
+    let reuseID: String = "FeedbackTableViewCell"
+
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = Constants.backgroundColor
+        // Do any additional setup after loading the view.
+        
+        isFeedback = setUpData() == 0 ? false : true
+        setUpNavigationBar()
+        setUpFeedbackLabel()
+        setUpConversationButton()
+        setUpFeedbackTableView()
+        setUpConstraints()
+        
+    }
+    
+    // TODO: hook up to actual server to load the data
+    // This is dummy data for testing
+    // Add variables to feedbackData to show the tableView + editing functionality
+    func setUpData() -> Int {
+        
+        let _ = Feedback(title: "Ithaca Transit Bug", message: "This app sometimes glitches out on me and shows the wrong bus times", hasUnread: true)
+        let _ = Feedback(title: "chatOS Idea", message: "what if admin responed via macOS applications and not iOS applications??", hasUnread: false)
+        feedbackData = []
+        return feedbackData.count
+        
+    }
+    
+    func setUpNavigationBar() {
+        
+        navigationController?.navigationBar.barTintColor = Constants.navigationTintColor
+        
+        let attributedTitle = NSAttributedString(string: "Feedback", attributes: [NSAttributedString.Key.font: UIFont(name: Constants.robotoMedium, size: 21)!, NSAttributedString.Key.foregroundColor: Constants.titleColor])
+        title = attributedTitle.string
+        
+        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(named: "pen"), style: UIBarButtonItem.Style.plain, target: self, action: #selector(handleNavigationBarRightTap))
+        
+    }
+    
+    func setUpFeedbackLabel() {
+        
+        feedbackLabel = UILabel()
+        feedbackLabel.lineBreakMode = .byWordWrapping
+        feedbackLabel.numberOfLines = 0
+
+        // format the alignment and spacing between lines
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.alignment = .center
+        paragraphStyle.lineSpacing = 10
+        
+        // format the text with specific color, font, size
+        let title = "No Feedback Yet\n"
+        let subtitle = "See feedback conversations here"
+        
+        let attributedText = NSMutableAttributedString(string: title, attributes: [NSAttributedString.Key.font: UIFont(name: Constants.robotoMedium, size: 17)!, .paragraphStyle: paragraphStyle, NSAttributedString.Key.foregroundColor: Constants.titleColor])
+        
+        attributedText.append(NSAttributedString(string: subtitle, attributes: [NSAttributedString.Key.font: UIFont(name: Constants.robotoRegular, size: 13)!, NSAttributedString.Key.foregroundColor: Constants.subtitleColor]))
```

Comment:
same comments about splitting these across multiple lines and moving the fonts into a `UIFont+Extensions.swift` file